src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -829,14 +829,14 @@ def _test_flexible_server_public_access_mgmt(self, database_engine, resource_gro
                  .format(database_engine, resource_group, servers[0], 'all'),
                  checks=[JMESPathCheck('resourceGroup', resource_group), JMESPathCheck('skuname', sku_name),
                          StringContainCheck('AllowAll_'),
-                         JMESPathCheck('host', '{}.{}.database.azure.com'.format(servers[0], database_engine))])
+                         StringContainCheck(servers[0])])
 
         # Case 2 : Provision a server with public access allowing all azure services
         self.cmd('{} flexible-server create -g {} -n {} --public-access {}'
                  .format(database_engine, resource_group, servers[1], '0.0.0.0'),
                  checks=[JMESPathCheck('resourceGroup', resource_group), JMESPathCheck('skuname', sku_name),
                          StringContainCheck('AllowAllAzureServicesAndResourcesWithinAzureIps_'),
-                         JMESPathCheck('host', '{}.{}.database.azure.com'.format(servers[1], database_engine))])
+                         StringContainCheck(servers[1])])
```

Comment:
why not check suffix any more?

Reply:
Its the actually just the server name that we want to validate. Removed the domain name because once we run it in the pipeline for the Staging env,the domain name will change. The domain name in stage is :azuredbpaccessm3smp5.orcabrci-seas1-a.mscds.com  and in production is : azuredbpaccessm3smp5.mysql.database.azure.com. As a result hardcoding the domains name in the validation would always fail it in the Stage env